RELEASE_NOTES.md

```diff
@@ -1,3 +1,21 @@
+### 0.11.3 January 12 2022 ####
+
+* Add deserialization whitelisting feature [#281](https://github.com/akkadotnet/Hyperion/pull/281)
+
+We've expanded our deserialization safety check to block dangerous types from being deserialized. You can now create a custom deserialize layer type filter programmatically: 
+
+```c#
+var typeFilter = TypeFilterBuilder.Create()
+    .Include<AllowedClassA>()
+    .Include<AllowedClassB>()
+    .Build();
+var options = SerializerOptions.Default
+    .WithTypeFilter(typeFilter);
+var serializer = new Serializer(options);
+```
+
+For complete documentation, please read the [readme on whitelisting types.](https://github.com/akkadotnet/Hyperion#whitelisting-types-on-deserialization)
```

Comment:
Change the link text to say "readme on filtering types for secure deserialization"